### ISSUE TITLE
Update Angular and RxJS

### ISSUE THREAD

**Issue by abdatta** [DesertFoxNV/ngx-autosize-input#7]
Please update the Angular and RxJS dependencies to their latest versions.
Installation gives the following warning: 
```
warning " > ngx-autosize-input@2.1.1" has incorrect peer dependency "@angular/core@^4.0.0".
warning " > ngx-autosize-input@2.1.1" has incorrect peer dependency "rxjs@^5.1.0".
```

**Comment by joshuawwright** [comment]
@abdatta I am not able to replicate this error. Could you please provide me with steps to replicate this error.  Here is what I tried:
1. Created a new project using angular cli (6.1.5)
2. Installed module using npm install ngx-autosize-input@2.1.1
3. Added and input box with the directive.
4. Tried serving and building.

When does the warning occur?

**Comment by abdatta** [comment]
It is a warning shown when I install some other module in my project (which already has ngx-autosize-input installed), using yarn/npm.  It is not a big issue, all it requires is to update your angular and rxjs to their latest versions.

**Comment by joshuawwright** [comment]
@abdatta This should be fixed in 2.2.1. Let me know if you still get the warning.
